code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -377,6 +383,21 @@ GLOBAL_LIST_INIT(wood_recipes, list ( \
 /obj/item/stack/sheet/mineral/wood/fifty
 	amount = 50
 
+/obj/item/stack/sheet/mineral/wood/attack(mob/living/carbon/human/H, mob/user)
```

Comment:
This should use `interact_with_atom`

Reply:
will do, i just based it off the cable code for healing robo limbs

Reply:
donezo :)